### PR TITLE
Fix windshaft filtering

### PR DESCRIPTION
**Pull request by Jesus89**
Fixes https://github.com/CartoDB/carto-vl/issues/1366

### CODE REVIEW

**Comment by elenatorro** [comment]
Good catch! 💪

File: test/unit/client/windshaft.test.js [hunk 1]
```diff
@@ -1,3 +1,138 @@
...
+    describe('_getServerInfoFrom', () => {
+        it('should convert viz into vizInfo', () => {
+            const viz = new Viz(vizMock);
+            const vizInfo = client._getServerInfoFrom(viz);
+            expect(vizInfo).toEqual(vizInfoMock);
+        });
+    });
+
+    describe('_needToInstantiateMap', () => {
+        it('should detect that it requires instantiation', () => {
```
Comment (VictorVelarde) [inline test/unit/client/windshaft.test.js]: It would be nice to test again, so we see no need for a new instantiation

Comment (VictorVelarde) [reply test/unit/client/windshaft.test.js]: And maybe a third change, altering one of the relevant properties. So it needs a new inst. again... Something so understand better the implement. from the test

**Comment by VictorVelarde** [comment]
Great @Jesus89 	I think we can 🚀

**Pull request merged**